website/install.md

```diff
@@ -21,14 +28,25 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
     </td>
     <td>
       <ul>
+        <li>On Arch Linux, install <code><a href="https://aur.archlinux.org/packages/git-town/">git-town</a></code> from the AUR</li>
```

Comment:
The problem is that each person has their own helper programs like cower, pacaur, yaourt, etc... that help them install programs from the aur and each of them have their own commands. 
All of them are wrappers around the basic commands to download the tarball from the website, run `tar xvf tarball.tar.gz`, then `makepkg -s` and finally `pacman -U /path/to/tar.xz` (the last two commands can be optionally combined into one). Would you like me to add these commands?